android/src/main/java/com/mr/flutter/plugin/filepicker/FileUtils.java

```diff
@@ -243,6 +243,21 @@ public static String getFullPathFromTreeUri(@Nullable final Uri treeUri, Context
         }
     }
 
+    @Nullable
+    @SuppressWarnings("deprecation")
```

Comment:
Do we really need to use deprecated SDKs? I'd like to merge it but I'm afraid I don't want to introduce more deprecated SDKs.